src/__test__/components/data-processing/classifier/Classifier.test.jsx

```diff
@@ -12,16 +12,24 @@ import CalculationConfig from '../../../../components/data-processing/Classifier
 import initialExperimentState from '../../../../redux/reducers/experimentSettings/initialState';
 
 import { initialPlotConfigStates } from '../../../../redux/reducers/componentConfig/initialState';
+import generatePlotUuid from '../../../../utils/generatePlotUuid';
```

Comment:
This should be renamed to something like `generateDataProcessingPlotUuid` -- we do not use this particular function for any other plots and tables.

Reply:
fixed in 92798de